integrations/crowclaw/README.md

```
@@ -34,26 +46,59 @@ Either way the definition is persisted under `~/.crowclaw/`. `mw-mcp` must be on
+> cause or a saved lesson. Once you've figured out why something failed or how a
+> fix worked, use `remember` to save that conclusion.
+
+## Verify
+
+```bash
+command -v mw-mcp
+```
+
+Then confirm tools were discovered:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)